components/exhibition/ExhibitionChaptersNavigation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="my-3 d-flex justify-content-end">
+    <b-link
+      v-if="previousChapter"
+      :to="previousChapter.url"
+      class="chapter-nav prev"
+      data-qa="previous chapter button"
+    >
+      {{ previousChapter.name }}
+    </b-link>
+    <b-link
+      v-if="nextChapter"
+      :to="nextChapter.url"
+      class="chapter-nav next"
+      data-qa="next chapter button"
+    >
+      {{ nextChapter.name }}
+    </b-link>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: {
+      exhibitionIdentifier: {
+        type: String,
+        required: true
+      },
+      chapterNavigation: {
+        type: Array,
+        required: true
+      }
+    },
+    data() {
+      return {
+        currentChapter: this.$route.params.chapter
+      };
+    },
+    computed: {
+      currentChapterIndex() {
+        if (this.chapterNavigation.length <= 1) return false;
```

Comment:
This does not do what it is intended to, because `false + 1 // => 1` and `false - 1 // => -1`.

Remove this line and nothing breaks, even on exhibitions with only a single chapter.